tests/unit/DocBlock/Tags/MethodTest.php

```diff
@@ -489,18 +523,18 @@ public function testCreateWithoutReturnType() : void
         $resolver           = new TypeResolver();
         $context            = new Context('');
 
-        $description = new Description('');
+        $description = new Description('My Description');
```

Comment:
Before the fix, a description after just a method with empty parentheses would cause the method name to be 'My".

I can change it to include both with and without description.

Reply:
Reverted. The unit tests should cover everything still.